src/components/shared/courses.js

```diff
@@ -8,10 +8,10 @@ function Courses ({ courseData, courseNotes, headingLevel }) {
 	var courseList="";
 	var trackCourseLevel="";
 	var coursesDisplay="<section class='row row-with-vspace'>";
-	var columnSetting = ((courseNotes === null || courseNotes === "") || (courseData === null || courseData === undefined)) ? "col-lg-12" : "col-lg-6";
+	var columnSetting = ((courseNotes === null || courseNotes === "") || (courseData === null || courseData === undefined || courseData.length === 0)) ? "col-lg-12" : "col-lg-6";
 	
```

Comment:
```suggestion
	var columnSetting = (!courseNotes || !courseData || !courseData.length) ? "col-lg-12" : "col-lg-6";
```

Suggested tweak to make the code shorter - see if it works